deep_groove_project/src/classification_dataset1/parameter_study.py

```diff
@@ -0,0 +1,94 @@
+from cls_set1 import cls_set1
+from cls_set1 import show_resultes
+import matplotlib.pyplot as plt
+import time
+
+
+def main():
+
+    params = {'batch_size': 20,
+              'shuffle': True,
+              'learning_rate': 1e-7,
+              'epochs': 200}
+
+    parameter_batch_size(params, output={"cli": False, "plot": False})
+    parameter_learning_rate(params, output={"cli": False, "plot": False})
+    parameter_epochs(params, output={"cli": False, "plot": False})
+
+
+def parameter_epochs(params, output):
```

Comment:
Could you add a doc string to your functions ? This would simplify my work to review your code.

Reply:
https://www.programiz.com/python-programming/docstrings